clj/resources/static_content/errors/504.html

```diff
@@ -0,0 +1,41 @@
+            <p><code>Error 504 - Gateway Time-out</code></p>
+            <h2>SlipStream didn't replied in time.<br/>It is probably overloaded.<br/>Please try again later.</h2>
```

Comment:
This needs to change: `didn't replied` to `didn't reply` in all the places of the commit.